tests/unit/components/todo-input/TodoInput.spec.js

```diff
@@ -0,0 +1,9 @@
+import { shallowMount } from '@vue/test-utils'
+import TodoInput from '@/components/todo-input/TodoInput.vue'
+
+describe('TodoInput component tests', () => {
+  const wrapper = shallowMount(TodoInput)
+  it('should match snapshot', () => {
+    expect(wrapper.element).toMatchSnapshot()
+  })
```

Comment:
Please add the following tests:
* template (check there is an input, and the input has the clear, save, checkbox icon)
* events (when clicking in the clear icon the text is empty, when clicking in save, checkbox icon events are emitted)